src/components/ReportActionItem/IOUAction.js

```diff
@@ -68,4 +98,10 @@ export default withOnyx({
     chatReport: {
         key: ({chatReportID}) => `${ONYXKEYS.COLLECTION.REPORT}${chatReportID}`,
     },
+    session: {
+        key: ONYXKEYS.SESSION,
+    },
+    personalDetails: {
```

Comment:
While this is a valid way to use Onyx, for performance reasons we have moved certain Onyx keys over to using the React Context API. See [`OnyxProvider`](https://github.com/Expensify/App/blob/aab34470f0c9c818774d7766dcd50d0dfb88611a/src/components/OnyxProvider.js), and in particular [`withPersonalDetails`](https://github.com/Expensify/App/blob/aab34470f0c9c818774d7766dcd50d0dfb88611a/src/components/OnyxProvider.js#L41).

For an example of how to use `withPersonalDetails`, look at the [`ReportActionItemSingle`](https://github.com/Expensify/App/blob/e88b7f984ab4e89e33844260c3a334622bf23023/src/pages/home/report/ReportActionItemSingle.js#L101) component.

Reply:
Updated with `withPersonalDetails`